.gitignore

```diff
@@ -1 +1,2 @@
 etc/ssl/
+.workspace
```

Comment:
this seems unnecessary

Reply:
Don you use vscode? ok, got it...

Reply:
I have 4 different editors and I'm against the idea of including them to each project gitgnore/hgignore. For such cases it's useful to have `~/.gitignore`, which hides all unnecessary stuff in all projects you're working with, something like this:
```
➜  ~  cat ~/.gitignore
.vscode
.idea
.DS_Store
```